src/api/hooks/lecture/useLecturePlan.ts

```
@@ -0,0 +1,142 @@
+    error,
+    refetch,
+  };
+};
+
+/**
+ * Hook to merge fetched lecture plan with existing data
+ * Prioritizes fetched data, falls back to courseData, then defaultData
+ */
+export const useMergedLectureData = (
+  fetchedData: Partial<LectureDetail> | null,
+  courseData: any | undefined,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)